tests/test_connections/test_local/test_search_services.py

```diff
@@ -136,101 +156,6 @@ def teardown_class(cls):
         cls.mailbox1.disconnect()
 
 
-# class TestFilteredSearchResult:
```

Comment:
We should uncomment this and add it back in.

Reply:
Done.